scripts/plugins/extauth-hook-AD.py

```diff
@@ -168,35 +175,40 @@ def _format_item(self, item):
         # CA-363207: Pam cannot handle space, put it inside []
         space_replacement = "+"
         if space_replacement in item:
-            raise ValueError("{} is not permitted in subject name".format(space_replacement))
+            raise ValueError(
+                "{} is not permitted in subject name".format(space_replacement))
 
         if " " in item:
             if self._backend == ADBackend.bd_pbis:
```

Comment:
This is a key part of the logic. I would like it if the backends were handled with more symmetry and the code stay flat:

```
if not (" " in item):
  return item

if self._backend == ADBackend.bd_pbis:
   return item.replace(" " , space_replacement)
else
   return "[{}]".format(item)
```

Reply:
just updated, please help to have a check